client/tsconfig.json

```diff
@@ -2,7 +2,7 @@
 	"compilerOptions": {
 		"target": "es2016",
 		"module": "es6",
-		"strict": true,
+		"strict": false,
```

Comment:
what does this do?

Reply:
This is to remove all the non type safe warning when calling methods on types. Example Phaser.scene may not have a certain method f but BattleRoom has, so need to suppress the warning that arise from scene.f() when the scene is used as an argument in another method.